app/memorize.py

```diff
@@ -8,14 +8,6 @@
 from app.memcached import memcached
 
 
-class CacheMissError(RuntimeError):
-    """
-    Exception thrown when memorized object is not found in cache and
-    error_on_miss is set to True.
-    """
-    pass
-
-
```

Comment:
Sorry, what's missing from it? The only change is that I removed `error_on_miss`, since the only place we'd want to use it is in the actual request/response cycle.